SU-SWS/stanford_bean_types
# Soe redesign

# NOT READY FOR REVIEW

# Summary
- This PR is for the call-to-action blocks. This is for the 3.x branch, and SoE needs the 2.x branch. See the [redesign-cjw](https://github.com/SU-SWS/stanford_bean_types/tree/redesign-cjw) for the 2.x branch.

# Needed By (Date)


# Urgency


# Steps to Test

1. Enable "Stanford Bean Call-to-Action" and "Stanford Bean  Call-to-Action Administration" module
2. Create a  Call-to-Action Bean
3. Verify it displays properly in the mange content view
4. Place block and verify display and responsive behavior


# Affected Projects or Products


# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-1651

- Anyone who should be notified? @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
